test/e2e/metallb/metallb.go

```diff
@@ -62,7 +65,7 @@ func Delete(metallb *metallbv1beta1.MetalLB) {
 	}, DeployTimeout, Interval).Should(BeTrue())
 }
 
-func Get(operatorNamespace string, useMetallbResourcesFromFile bool) (*metallbv1beta1.MetalLB, error) {
+func Get(operatorNamespace string, useMetallbResourcesFromFile bool, opts ...MetalLBOption) (*metallbv1beta1.MetalLB, error) {
```

Comment:
I know it was alread there, but this feels more like a "New" as we are not getting it from anywhere.